src/components/GrantRequest/AddModal.css

```diff
@@ -0,0 +1,26 @@
+.AddModal {
+  max-width: 520px !important;
+}
+
+.AddModal .content {
+  margin-bottom: 0 !important;
+}
+
+.AddModal__Title {
+  font-style: normal !important;
+  font-weight: 600 !important;
+  font-size: 17px !important;
```

Comment:
Wouldn't it be better to use more specific selectors instead of using !important on each property?

For example: `.GovernanceContentModal .AddModal .AddModal__Title`

Reply:
`!important` is used as this className is used in "decentraland-ui"'s Modal directly.